PyOpenWorm/context.py

```diff
@@ -162,19 +156,13 @@ def save_imports(self, graph=None):
                 ctx.save_imports(graph)
         return graph
 
-    def save_context(self, graph=None, inline_imports=False, autocommit=True, seen=None):
+    def save_context(self, graph=None, inline_imports=False, autocommit=True, seen=set([])):
```

Comment:
That's not how we default mutable types. This way will prevent saving a context again (i.e., to a different graph or after a change) unless you provide an explicit `seen` argument.